app/assets/stylesheets/lux/_show_heading.scss

```diff
@@ -0,0 +1,8 @@
+@import "lux/variables";
+
+div#document.document > div > h1 {
```

Comment:
Could you add a class to the element that this is referring to and define the rule that way?

Reply:
This has been addressed and in the next commit.